### PR TITLE
Fix linter warnings

### PR DESCRIPTION
**Pull request by croumegous**
Fixed all warnings in `next build/next lint`

Added a new npm command: `npm run fix` to run format and lint code.
To make review easier I have made 2 commits, the first one adds eslint plugins to automatically handle import sort and unused import.
And a second commit which fixes the remaining warning manually, so this commit is more "opinionated" and needs more attention during the review.
(I have added some `//eslint-disable-next-line` for some part, where it seems to be the best option)

(There is a lot of changes so it might create conflicts for other people working on /website)

### CODE REVIEW

File: website/.eslintrc.json [hunk 1]
```diff
@@ -6,6 +6,19 @@
...
+    "unused-imports/no-unused-vars": [
+      "warn",
+      {
+        "vars": "all",
+        "varsIgnorePattern": "^_",
+        "args": "after-used",
+        "argsIgnorePattern": "^_"
+      }
+    ]
+  },
```
Comment (AbdBarho) [inline website/.eslintrc.json:22]: I know that customizing eslint is always very opinionated... imo there is no need for these rules.

I will have to wait for opinions of other team member...

**Review by AbdBarho — COMMENTED** [review]
@croumegous thanks for the contribution!

could you please rebase? easier to review.

With the exception of my comment it looks fine.

File: website/.eslintrc.json (hunk 1 above)
Comment (croumegous) [reply website/.eslintrc.json]: I've changed it to keep the less rule as possible to make it work:
```
"unused-imports/no-unused-imports": "warn",  //mandatory to automatically remove unused import  
"simple-import-sort/imports": "warn", //mandatory to automatically sort import  
"simple-import-sort/exports": "warn"  //mandatory to automatically sort export
```

Comment (fozziethebeat) [reply website/.eslintrc.json]: I think these three small additions look good.

**Review by fozziethebeat — APPROVED** [review]
Things look a lot cleaner now, thank you~

**Comment by fozziethebeat** [comment]
Please fix the merge issues and then I'll re-review

**Pull request merged**